InvaderZim85/MsSqlToolBelt
# "Hide SQL Query" fix

## Whats changed?

- Fixed the "Hide SQL Query" option (*Settings > Class generator options*)
- Performance optimization (Load / Save process of the settings)

## Related issues

#186 

## How to test?

- Start the program
- Change the option "Hide SQL Query" to `true` (checked)
- Save the change
- Restart the program
- The *SQL Query* region (Class generator) should still be hidden

## Checklist

Complete the following checklist to make sure you haven't forgotten anything

- [x] Unnecessary comments have been removed
- [x] TODOs have been edited or provided with appropriate information
- [x] Usings have been cleaned up
- [x] All required files have been committed
- [x] Code has been reviewed by the developer
